src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { User, UserInfo } from '../UserInfo';
+
+export type Todo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null,
```

Comment:
you can make this field optional 
```suggestion
  user?: User,
```